SETUP/manual_web/split_test/vertical_horizontal_split_flex.php

```diff
@@ -0,0 +1,43 @@
+<?php
+$relPath="../../../pinc/";
+include_once($relPath.'base.inc'); // require_login()
+include_once($relPath.'slim_header.inc');
+
+// Demonstrate two pane vertical split
+
+require_login();
+
+$title = _('Vertical and Horizontal Split with Flex Layout');
+
+$js_files = [
+    "$code_url/scripts/splitControl.js",
+    "v_h_split.js",
+];
+
+// overflow: hidden prevents getting incorrect container size
+// when using flex layout and window size is reducing
+$header_args = [
+    "js_files" => $js_files,
+    "body_attributes" => "style='margin: 0; overflow: hidden'",
+];
+
+slim_header("$title", $header_args);
+
+echo "<div style='display: flex; flex-direction: column; height: 100vh;'>\n";
+
+echo "<div style='background-color: aquamarine;'>";
+echo "<h1>$title</h1>\n";
+echo "</div>\n";
+
+echo "<div id='top-container' style='background-color: beige; flex: auto;'>\n";
+
+echo "<div>This is pane1</div>\n";
```

Comment:
Please include some more content in each pane. You might just throw the text from `/horizontal_split_flex.php` into a .txt file in the directory and `file_get_contents()` it into each pane in each test.

Reply:
done this with text and image